src/base/Module.zig

```diff
@@ -70,16 +73,16 @@ pub const Store = struct {
     pub fn lookup(
         self: *Store,
         name: []const u8,
-        package_shorthand: ?[]const u8,
+        package_shorthand: ?[]u8,
     ) ?Idx {
         const items = self.modules.items;
-
         for (0..self.modules.len()) |index| {
-            const other_name = items.items(.name_segments)[index];
-            if (name == other_name) {
-                const other_package_shorthand = items.items(.package_shorthand)[index];
-                if (other_package_shorthand == package_shorthand) {
-                    return @enumFromInt(@as(u32, index));
+            const item = items.get(index);
+            if (std.mem.eql(u8, name, item.name)) {
+                if (package_shorthand == null and item.package_shorthand == null) {
+                    return @enumFromInt(@as(u32, @intCast(index)));
+                } else if (package_shorthand != null and item.package_shorthand != null and std.mem.eql(u8, package_shorthand.?, item.package_shorthand.?)) {
```

Comment:
super nitpick: might want some newlines to break up this line around the `and` operators